shell/platform/common/cpp/BUILD.gn

```diff
@@ -30,12 +30,28 @@ source_set("common_cpp_library_headers") {
   configs += [ ":desktop_library_implementation" ]
 }
 
+source_set("common_input_cpp") {
```

Comment:
Nit: I would call this `common_cpp_input` since `common/cpp` is the directory, and input is a subset of that. (There are other `common` directories in the tree.)